### ISSUE TITLE
Signing Up Via Permalink Page Experience Breaks

### ISSUE THREAD

**Issue by namimody** [DoSomethingArchive/legacy-website#5585]
Happening from most browsers (Chrome, IE, Android, Safari/iOS) and especially via Japan (but problems reported from Brazil and US).

Users who try to sign up for a campaign from a permalink or pitch page are 404'd, brought to their "account information page", or page shows up in barebones HTML:
![image](https://cloud.githubusercontent.com/assets/12396076/10673084/f21b9590-78c1-11e5-8b68-2413677b117e.png)

**Comment by angaither** [comment]
@namimody anon or auth users or both?

**Comment by namimody** [comment]
@angaither ahh would need to test auth users. One sec.

**Comment by namimody** [comment]
@angaither Seems to only be the case for anon users.

**Comment by angaither** [comment]
... i bet pitch page sign up button is also broken for them too (it's the same code)

**Comment by namimody** [comment]
Doesn't work for me or @mikefantini on node 5740 (Out of the Closet Cards on Thor)

**Comment by namimody** [comment]
I'm not able to access 1 Star for Hate (node 3501) after creating an account or by using an existing non-admin) account on Thor @angaither.

Tried to log in using existing account, and saw this:

![image](https://cloud.githubusercontent.com/assets/12396076/10675319/7a13d60a-78cd-11e5-96cb-d49222d33635.png)

**Comment by angaither** [comment]
having trouble replicating this locally

**Comment by angaither** [comment]
@namimody I was able to get redirected to my user profile page when
- anon user (previously exists) 
- sign up for a campaign from a permalink that is unpublished

but I haven't been able to get the `an error has occured` page

**Comment by mikefantini** [comment]
I just got the `an error has occurred` page by going to:
- https://thor.dosomething.org/us/campaigns/level-playing-field as anon user
- Clicking Sign Up button
- Logging in with an account that exists (uid 2803726)

![image](https://cloud.githubusercontent.com/assets/6330971/10706287/a165c358-79b3-11e5-88b2-9f69001f4670.png)

**Comment by mikefantini** [comment]
I just got the same experience on the same campaign when I went to create an account rather than log in to an existing one (uid 3118804)

**Comment by mikefantini** [comment]
I was able replicate this as well on 1 Star for Hate by:
- going to thor.dosomething.org as an anonymous user
- clicking on the Sign Up button
- logging in as user 2803726
- then I get the following error:

![image](https://cloud.githubusercontent.com/assets/6330971/10736652/945e922e-7be4-11e5-9b66-57941c7ecd3e.png)

**Comment by angaither** [comment]
http://dev.dosomething.org:8888/us/campaigns/school-standstill

```
EntityMetadataWrapperException: Unknown data property field_description. in EntityStructureWrapper->getPropertyInfo() (line 335 of /var/www/dev.dosomething.org/html/profiles/dosomething/modules/contrib/entity/includes/entity.wrapper.inc).
The website encountered an unexpected error. Please try again later.
```

**Comment by angaither** [comment]
that field is on action guide content

**Comment by angaither** [comment]
yeah, confirmed, all the campaigns that are failing it's because of the action guide that's attached to it.

what's strangest to me, is how it isn't failing on the full node view for admins

**Comment by angaither** [comment]
non-admin `dpm($vars['field_action_guide']);`
![screen shot 2015-10-26 at 2 44 07 pm](https://cloud.githubusercontent.com/assets/645205/10738914/0d8f56aa-7bf0-11e5-958d-6a5c1ef3306e.png)

admin `dpm($vars['field_action_guide']);`
![screen shot 2015-10-26 at 2 44 40 pm](https://cloud.githubusercontent.com/assets/645205/10738922/1e6dcca4-7bf0-11e5-9989-4cec7dc55e7d.png)

note the non-admin has an additional LANGUAGE_NONE array wrapped around it
the question is why?
those fields/content hasn't been touched by this translation stuff

**Comment by angaither** [comment]
it's easy enough to check if the array key isn't numeric, but I just don't understand why this would have changed for non-admins

**Comment by angaither** [comment]
fixed with #5631
